examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -382,18 +385,31 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # mlflow initial
+    experiment_id = mlflow.create_experiment('translateion-{}'.format(model_args.model_name_or_path))
```

Comment:
According to what you've mentioned in your Daily Scrum, if what I think is right, we should be still able to use MLFlow Tracking just using Accelerate instance. Then do we not need to add all these repetitive MLFlow APIs for each task?